rootfs/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update \
 		golang \
 		make \
 		--no-install-recommends \
-	&& export DOCKER_REGISTRY_TAG=minio \
+	&& export DOCKER_REGISTRY_TAG=master \
```

Comment:
Can't we use the `v2.4.1` tag here, for example, instead of `master`? Or at least some tag?